templates/web/smidsy/index.html

```diff
@@ -45,25 +45,6 @@ <h2>Need cycling collision data?</h2>
     </div>
   </div>
 
-  <div class="front__why">
-    <div class="container">
-      <h2>Why use Collideoscope?</h2>
-      <div class="squidgy-quote squidgy-quote--blue squidgy-quote--left">
-        <img src="/cobrands/smidsy/images/user-amy.jpg" width="80" height="80" alt="">
```

Comment:
May as well `git rm cobrands/smidsy/images/user-*` while you’re at it. We don’t use these images anywhere else.